core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -366,16 +367,16 @@ class BrokerToControllerRequestThread(
         val controllerAddress = activeControllerAddress()
         if (controllerAddress.isDefined) {
           requestIter.remove()
-          return Some(RequestAndCompletionHandler(
+          return util.Collections.singletonList(new RequestAndCompletionHandler(
             time.milliseconds(),
             controllerAddress.get,
             request.request,
-            handleResponse(request)
+            response => handleResponse(request)(response)
           ))
         }
       }
     }
-    None
+    util.Collections.emptyList()
```

Comment:
I had the exact same thought - getting rid of the `util` prefixes in the various places in Scala classes where Java collection types are now used.
Unfortunately IntelliJ flagged the `Collection` return type of `generateRequests()` with a `Reference must be prefixed` warning, so as I didn't find a nice way to avoid the prefix there, I decided it's preferable to just keep the prefixes for all the related usages of Java collection types.